docs/development-update-22.44.md

```diff
@@ -0,0 +1,120 @@
+# Development Update 22.44
+
+## Overview
+
+This is an interim update ("code drop") to give the validation team
+an opportunity to take the DPDK code out for a test drive.
+
+## Project Name
+
+Note that the P4-OVS Split Architecture has a new name: **P4 Control Plane**.
+
+## What's Changed
+
```

Comment:
Should we add a section for Feature Support specifying the following:
1. OpenConfig gnmi support
2. P4Runtime Support

This will give Validation clear idea on what they need to test with this drop and also to external user who is trying this drop?

Reply:
This isn't new. What information would we give them that is specific to DPDK?

Reply:
OpenConfig GNMI support
•	Port Configuration for following port types (SET/GET)
o	Virtual-device: VIRTIO_NET, TAP
o	Physical-device: LINK
•	Hotplug support for VIRTIO_NET ports to qemu based VM

P4Runtime support
•	Set the forwarding pipeline
•	Configure forwarding rules (Supported match types are Exact, LPM, Ternary)